web-api/src/persistence/postgres/workitems/getDocumentQCInboxForUser.ts

```diff
@@ -63,14 +65,80 @@ export const attachDocketEntriesToWorkItemQC = async ({
     entryByCompositeKey.set(key, entry);
   }
 
+  const consolidatedCaseData = await getConsolidatedCaseData({ workItems });
+
   const workItemsWithDocketEntries = workItems.map(w => {
     const key = `${w.docketNumber}|${w.docketEntryId}`;
     const docketEntry = entryByCompositeKey.get(key);
+
+    const consolidatedInfo = w.leadDocketNumber
+      ? consolidatedCaseData.get(w.leadDocketNumber)
+      : undefined;
+
     return {
       ...fromKyselyWorkItemAndCase(w),
       docketEntry: docketEntry ?? ({} as RawDocketEntry),
+      consolidatedCases: consolidatedInfo,
     };
   });
 
   return workItemsWithDocketEntries;
 };
+
+const getConsolidatedCaseData = async ({
+  workItems,
+}: {
+  workItems: WorkItemWithCaseInfoKysely[];
+}) => {
+  const leadDocketNumbers = Array.from(
+    new Set(
+      workItems.filter(w => w.leadDocketNumber).map(w => w.leadDocketNumber!),
+    ),
+  );
+
+  if (leadDocketNumbers.length === 0) {
+    return new Map();
+  }
+
+  const consolidatedCases = await getDbReader(reader => {
+    return reader
+      .selectFrom('dwCase as c')
+      .select(['c.docketNumber', 'c.docketNumberSuffix', 'c.leadDocketNumber'])
+      .where(eb =>
+        eb.or([
+          eb('c.leadDocketNumber', 'in', leadDocketNumbers),
+          eb('c.docketNumber', 'in', leadDocketNumbers),
```

Comment:
The `leadDocketNumber` should also be on the lead case itself as `leadDocketNumber`